.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.yarn/
```

Comment:
The gitignores are slightly more complicated than this — have you seen https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored? We have typically used the list provided in the non-zero install section, e.g.: https://github.com/MetaMask/metamask-module-template/blob/62bb1756ef6458580450a656dd7498c1610e8bcb/.gitignore#L70

Reply:
Thanks for the pointers on aligning the project yarn config - hope i got it right now.